src/model/enhance-numbered-lists.ts

```diff
@@ -167,6 +167,65 @@ const addH3s = (elements: CAPIElement[]): CAPIElement[] => {
 	return withH3s;
 };
 
+const isFullReviewLink = (element: CAPIElement): boolean => {
+	if (!element) return false;
+	// Checks if this element is a 'full review' based on the convention: <ul> <li><p><strong>Full review:</strong> <a href="https://www.theguardian.com/technology/2019/oct/22/oneplus-7t-pro-review-the-best-kind-of-deja-vu">OnePlus 7T Pro review: the best kind of deja vu</a></p></li> </ul>
+	if (
+		element._type !== 'model.dotcomrendering.pageElements.TextBlockElement'
+	) {
+		return false;
+	}
+	const frag = JSDOM.fragment(element.html);
+	if (!frag || !frag.firstElementChild) return false;
+
+	const hasULWrapper = frag.firstElementChild.nodeName === 'UL';
+	const hasLINestedWrapper =
+		frag.firstElementChild.firstElementChild &&
+		frag.firstElementChild.firstElementChild.nodeName === 'LI';
+
+	const wrapper =
+		frag.firstElementChild.firstElementChild &&
+		frag.firstElementChild.firstElementChild.firstElementChild;
+
+	const hasPWrapper = wrapper && wrapper.nodeName === 'P';
+	const containsStrongTags =
+		wrapper && wrapper.outerHTML.includes('<strong>');
+	const containsLinks = wrapper && wrapper.outerHTML.includes('<a ');
+
+	const hasFullReviewText =
+		wrapper &&
+		wrapper.firstElementChild &&
+		wrapper.firstElementChild.textContent === 'Full review:';
```

Comment:
Is it certain that this text is always present?

Reply:
good point, no https://www.theguardian.com/world/2020/mar/25/coronavirus-key-questions-answers-everything-you-need-to-know-covid-19

Full review no longer makes sense as a component name